packages/react-components/src/organisms/__tests__/EventCard.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ComponentProps } from 'react';
+import { render } from '@testing-library/react';
+import { createEventResponse, createGroupResponse } from '@asap-hub/fixtures';
+
+import EventCard from '../EventCard';
+
+const props: ComponentProps<typeof EventCard> = {
+  ...createEventResponse(),
+  groups: [],
+  href: '',
+};
+
+it('Renders an event', () => {
+  const { getByRole } = render(<EventCard {...props} title="My Event" />);
+  expect(getByRole('heading', { level: 4 })).toHaveTextContent('My Event');
```

Comment:
4? o.O that's deep, no way it's used that deep down?
I also kinda feel like we should have a heading level react context or something lol